src/main/java/sungshin/sooon/controller/BoardController.java

```diff
@@ -0,0 +1,66 @@
+package sungshin.sooon.controller;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Sort;
+import org.springframework.data.web.PageableDefault;
+import org.springframework.data.domain.Pageable;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+import sungshin.sooon.dto.PostRequestDto;
+import sungshin.sooon.dto.PostResponseDto;
+import sungshin.sooon.dto.PostUpdateRequestDto;
+import sungshin.sooon.model.Post;
+import sungshin.sooon.service.BoardService;
+
+import java.security.Principal;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/api/v1/board")
+public class BoardController {
+
+    private final BoardService boardService;
+
+    // 전체 글 리스트
+    @GetMapping("/list")
+    public Page<Post> list(@PageableDefault(page = 1, sort = "created_at", direction = Sort.Direction.DESC) Pageable pageable, Model model) {
+        Page<Post> postList = boardService.getPostList(pageable);
+        model.addAttribute("postList", postList);
+
+        return boardService.getPostList(pageable);
+    }
+
+    // 게시글 저장
+    @PostMapping("/post")
+    public Long createPost(@RequestBody PostRequestDto postRequestDto) {
+        return boardService.savePost(postRequestDto);
+    }
+
+    // 게시글 상세
+    @GetMapping("/detail/{post_id}")
+    public PostResponseDto getBoardInformation(@PathVariable("post_id") Long post_id) {
+        return boardService.readPost(post_id);
+    }
+
+    // 게시글 수정
+    @PutMapping("edit/{post_id}")
```

Comment:
```suggestion
    @PutMapping("/{post_id}")
```

> 2️⃣ 자원에 대한 행위는 HTTP Method(GET, POST, PUT, DELETE)로 표현한다는 점
출처: https://devuna.tistory.com/79 [튜나 개발일기]